### ISSUE TITLE
[bug]

### ISSUE THREAD

**Issue by srinivasb99** [whoeevee/EeveeSpotify#415]
Lyrics colors are sometimes inconsistent and glitch out 

![image](https://github.com/user-attachments/assets/675c87e0-947f-4ace-8dd0-69362bad8f94)

Color randomly becomes the color shown in he picture

**Comment by Wenwenwhen** [comment]
If you mean the lyrics becoming white as the song goes on, until there's no gray lyrics left, that's a normal Spotify feature. The lyrics that have already been read are changed to a different color compared to the rest, indicating their completion.

**Comment by srinivasb99** [comment]
> If you mean the lyrics becoming white as the song goes on, until there's no gray lyrics left, that's a normal Spotify feature. The lyrics that have already been read are changed to a different color compared to the rest, indicating their completion.

No, I mean the lyrics background card color. It's an issue with the Spotify default colors, as after turning it off and using custom colors it fixes the issue. Turning on Spotify default colors causes the bug
